builtin/mainmenu/content/dlg_package.lua

```diff
@@ -205,11 +208,41 @@ local function get_formspec(data)
 
 	elseif current_tab == 2 then
 		local hypertext = info.info_hypertext.head .. info.info_hypertext.body
-
 		table.insert_all(formspec, {
 			"hypertext[0,0;", W, ",", tab_body_height - 0.375,
 			";info;", core.formspec_escape(hypertext), "]",
 		})
+	elseif current_tab == 3 then
+		if not package.reviews and not data.reviews_error and not data.reviews_loading then
+			data.reviews_loading = true
+
+			contentdb.get_package_reviews(package, function(reviews)
+				if not reviews then
+					data.reviews_error = true
+				end
+				ui.update()
+			end)
+		end
+
+		if package.reviews then
+			local hypertext = package.reviews.head .. package.reviews.body
+			hypertext = hypertext:gsub("<img name=\"?blank.png\"? ",
```

Comment:
What's the purpose of this image? Why does this text exist in the hypertext?
If it is needed, please add a short comment, else remove the code.